sdk/nodejs/src/request.ts

```diff
@@ -1,11 +1,21 @@
 import axios, { AxiosRequestConfig, AxiosRequestHeaders, AxiosResponse } from 'axios'
+import axiosRetry from 'axios-retry'
 import { SDKEventBatchRequestBody, DVCLogger } from '@devcycle/types'
 import { DVCPopulatedUser } from './models/populatedUser'
 import { DVCEvent, DVCOptions } from './types'
 
 const axiosClient = axios.create({
+    timeout: 60 * 1000,
     validateStatus: (status: number) => status < 400 && status >= 200,
 })
+axiosRetry(axiosClient, {
+    // will do exponential retry until axios.timeout
+    retryDelay: axiosRetry.exponentialDelay,
+    retryCondition: (error) => {
+        return (error?.response?.status || 0) >= 500
```

Comment:
this should also retry if there's no response on the error, otherwise this won't actually fix the problem (which is a request timeout)